server/lib/cron-scheduler/save-document.ts

```diff
@@ -59,7 +58,9 @@ export class SaveDocument {
             }
           }
         });
-        
+        }
+      }catch(error){
+        log(this.logPath, `Index '${index}' exists? false`, 'debug');
         log(this.logPath, `Status of create a new index: ${JSON.stringify(response)}`, 'debug');
```

Comment:
If the index pattern doesn't exist, the request doesn't throw an error, so it doesn't enter in the `catch` block.

The `response` variable is decalred in the `try` block. Should you move the log there.